src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: $main-font-famaly;
+}
+
+.contentInformacion {
+  margin: 0 16 16;
+  width: 166px;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
+  border-radius: 5px;
+  border: 1px;
```

Comment:
The 'border' property is redundant as it is immediately overridden by the next 'border' property. You can remove this line to avoid confusion.